feat(content): Adding some hails and news entries that provide hints about TMBR's mission requirements (#7963)
endless-sky/endless-sky: data/human/news.txt

			"No bread is better than half a loaf, because then the passengers won't fight over it."
			"A painting is worth a thousand words, because the profit margin on transporting visual art is much higher than on dry literature."
			"A journey of a thousand miles has too short of a distance to make any profit."

> 			"The other day I happened across a rock band whose transport flaked out. Good thing I had eight free seats for them, because it got me into a free concert at their next stop."

Reviewer: ```suggestion
			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
```
Bit more natural phrasing but also highlights specific number more directly.

> 			"The other day I happened across a rock band whose transport flaked out. Good thing I had eight free seats for them, because it got me into a free concert at their next stop."

Reviewer: ```suggestion
			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
```
Bit more natural phrasing but also highlights specific number more directly.

> 			"The other day I happened across a rock band whose transport flaked out. Good thing I had eight free seats for them, because it got me into a free concert at their next stop."

Reviewer: ```suggestion
			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
```
Bit more natural phrasing but also highlights specific number more directly.

> 			"The other day I happened across a rock band whose transport flaked out. Good thing I had eight free seats for them, because it got me into a free concert at their next stop."

Reviewer: ```suggestion
			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
```
Bit more natural phrasing but also highlights specific number more directly.

			"I got a raise today. Only two credits more than my previous salary. Ridiculous!"
			"Found a bomb in a crate the other day. Turns out I probably saved a very important diplomat from being blown to pieces."
			"Apparently, the spaceports in the Deep have fully automated loading systems. Imagine that!"

> 			"The other day I saw a band manager scrambling to find transport. She needed something with at least 8 free bunks, and most of the ships around at the time were already full."

Reviewer 2: ```suggestion
			"The other day I saw a band manager scrambling to find transport. She needed something with at least eight free bunks, and most of the ships around at the time were already full."
```

			"Some musicians think they are so sophisticated just because they play antique tunes. I prefer to innovate beyond what already exists, even if the result is different. Doesn't the art of music demand such creativity?"
			"Stories about extra-terrestrials are very unpopular now that most of humanity is living off-Earth."
			"Stories about alien lifeforms became a lot more popular after humanity actually met alien lifeforms."

> 			"It takes surprisingly little space to transport a band on a tour. Eight free bunks and about half a cubic meter of cargo each, so long as they are good at packing."

Reviewer 2: ```suggestion
			"It takes surprisingly little space to transport a band on a tour. Eight free bunks and about half a unit of cargo each, so long as they are good at packing."
```

			"You'll never believe what happened on my first day on the job. Someone tried to rob a building while I was in the parking lot. I had to ram their van to keep them from getting away!"
			"They make me fly around the planet for ridiculous things, but you don't hear me complain with the salary they pay me."
			"The vegetables are supposedly more fresh when I personally pick them up at the spaceport than when they get delivered through the 1-hour delivery service."

> 			"I used to have a job flying a band around, but they wanted to spend some time in the Dirt Belt, and I was hoping for work closer to the paradise worlds."

Reviewer 2: ```suggestion
			"I used to have a job flying a band around, but they wanted to spend some time in the Dirt Belt, and I was hoping for work closer to the Paradise Worlds."
```
